.github/workflows/test.yml

```diff
@@ -34,11 +34,14 @@ jobs:
           cmake -S test -B build/local
           cmake --build build/local
           cmake --build build/local --target test
-          sudo -E cmake --build build/local --target install
+          cmake --install build/local --prefix ./install_dir
 
       - name: test installed build
         run: |
-          cmake -S test -B build/installed -D TEST_INSTALLED_VERSION=1
+          sudo apt install fzf ripgrep
+          fzf --walker-root=./install_dir -f "lib /cmake/ /dependencyConfig.cmake" | rg lib | rg cmake | rg Config.cmake
+          fzf --walker-root=./install_dir -f header_onlyConfig.cmake | rg share/cmake
+          cmake -S test -B build/installed -DCMAKE_INSTALL_PREFIX=./install_dir -D TEST_INSTALLED_VERSION=1
```

Comment:
- This is a lot of changes in the install / test procedure. Why is the original build and install not enough?
- Tbh in CI I would prefer testing the global install (without arguments) to make sure the packages are correctly found in a standard system setup.

Reply:
I've addressed the issues mentioned, but tbh I don't have any idea why in logs it said  `unknown option: --walker-root=./install_dir` but afaik it's a working option for `fzf` , at least exists for `fzf` on my machine

Reply:
Probably a version issue. But do we really need `fzf` and `ripgrep`? Isn't the original test build + suite enough?

Reply:
Is enough?
Maybe no, because there's no else way to check if it works as expected
Do we need fzf and ripgrep?
`ripgrep` -- no, plain `grep` would be enough
`fzf` -- I just don't know what else can be that powerful. Maybe a really hard to understand `find` command can, though it needs investigation.

Reply:
To be honest, I'm also not quite sure I understand what the commands are checking for as I know neither of those tools, maybe adding a one or two line comment with an explanation would help. But to me it looks like you're checking for existence of some CMake config files in the share and lib directories, right? Could it just be done with a `test -e <path to header_onlyConfig.cmake>` in the share directory?

Imo the existence of that the file is in the share dir and the fact that CMake can successfully build the project should be a good enough check for now, what do you think?

Reply:
> To be honest, I'm also not quite sure I understand what the commands are checking for as I know neither of those tools, maybe adding a one or two line comment with an explanation would help. But to me it looks like you're checking for existence of some CMake config files in the share and lib directories, right? Could it just be done with a `test -e <path to header_onlyConfig.cmake>` in the share directory?

You are right. Also yes, maybe there should be some comments.
Maybe the `test -e /usr/share/cmake/header_only-1.0/header_onlyConfig.cmake` would be enough in the case.

> Imo the existence of that the file is in the share dir and the fact that CMake can successfully build the project should be a good enough check for now, what do you think?

You see, the problem is not if the project is buildable. The problem is if it's usable via `find_package`.
In one environment it would be ok. 

Now it's usable from our ubuntu-cmake 64 bit perspective.

 But now IDK if from 32 bit perspective 64 bit libs shouldn't be visible, so IDK should we make some tests if  libs installed to `/usr/lib` are not visible from 64 bits environment and `/usr/lib64` are not visible from 32 bits environment . Right now investigating if this should be strict.

Reply:
> But now IDK if from 32 bit perspective 64 bit libs shouldn't be visible, so IDK should we make some tests if libs installed to /usr/lib are not visible from 64 bits environment and /usr/lib64 are not visible from 32 bits environment . Right now investigating if this should be strict.

I think this is kind of out of the scope of this project and we can just rely on CMake to do the right thing assuming that we are using the API correctly.